app.py

```diff
@@ -20,8 +20,8 @@ def effectively_never():
 
 
 def end_of_day_on_weekdays():
-    """ Cron schedule that runs at 6pm EST (22:00 UTC) on weekdays. Used for deployments. """
-    return Cron('0', '22', '?', '*', 'MON-FRI', '*')
+    """ Cron schedule that runs at 11pm EST (03:00 UTC) on weekdays. Used for deployments. """
+    return Cron('0', '3', '?', '*', 'TUES-SAT', '*')
```

Comment:
I suppose others more used to these tuples might disagree, but `Cron` always seems to me like the poster child for why keyword args are so much better than positional ones, even for required arguments that could be positional. e.g.. (untested):
```suggestion
    return Cron(minutes='0', hours='3', day_of_month='?', month='*', day_of_week='TUES-SAT', year='*')
```
(This could also be done another time when there was time to do it consistently in all uses.)

And this also makes me wish for the calendar system that was once used by my cable company where they associated midnight to 6am with the previous day, dividing the day into 6:00a-11:59a, 12:00p-11:59p, 12:00x-5:59x, and allowing this still to be seen as a Mon-Fri schedule. But I think the dates used here expressed as TUE-SAT are fine; this doesn't really represent a change in daily schedule, just a moving later into the night.